deltachat-contact-tools/src/lib.rs

```diff
@@ -295,21 +295,26 @@ pub fn sanitize_name_and_addr(name: &str, addr: &str) -> (String, String) {
     (name, addr)
 }
 
-/// Normalize a name.
+/// Normalizes a name.
 ///
-/// - Remove quotes (come from some bad MUA implementations)
+/// - Removes newlines
+/// - Removes quotes (come from some bad MUA implementations)
 /// - Trims the resulting string
+/// - Removes potentially-malicious RTLO characters
 ///
 /// Typically, this function is not needed as it is called implicitly by `Contact::add_address_book`.
-pub fn normalize_name(full_name: &str) -> String {
-    let full_name = full_name.trim();
+pub fn normalize_name(name: &str) -> String {
+    let name = name.replace(['\n', '\r'], " ");
```

Comment:
Maybe replace "\r\n" with the only space as it is just a DOS-style newline?

Reply:
It doesn't seem to me that this would be worth even the tiny amount of extra code complexity: Users need to do really weird things to insert a line break here, and a double space is fine then - everything as long as the user interface isn't screwed up.

This PR is about making DC do sensible things when encountering very, very weird inputs.